### ISSUE TITLE
fix or remove flask.debug.client_cert_file parameter

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-ch#9]
Right now the Parameters.py file contains:
```
    {   
        'name': "flask.debug.client_cert_file",
        'val': "/home/mbrinn/.gcf/mbrinn-cert.pem",
        'desc': "Debug client cert file"
    },
```
Change that to something real if it's needed, or remove it if it's not.

 _Imported from trac ticket [#9](http://trac.gpolab.bbn.com/chapi/ticket/9),  created by chaos on 11-07-2013 at 17:02, last modified: 11-08-2013 at 12:42_

**Comment by tcmitchell** [comment]
This will get fixed as part of issue #1.

_Trac comment by tmitchel (github user: tcmitchell) on 11-07-2013 at 17:17_

**Comment by tcmitchell** [comment]
The value has been changed to an example value in the sample config file and in Parameters.py. The config variable still exists so that developers can configure an appropriate certificate as necessary via the config file.

_Trac comment by tmitchel (github user: tcmitchell) on 11-08-2013 at 12:42_
